script.sleep/default.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+import xbmc
+import xbmcgui
+import xbmcaddon
+
+
+__addon__ = xbmcaddon.Addon(id='script.sleep')
+__cwd__ = __addon__.getAddonInfo('path').decode("utf-8")
+__resource__ = xbmc.translatePath(os.path.join(__cwd__, 'resources').encode("utf-8")).decode("utf-8")
+__lib__ = xbmc.translatePath(os.path.join(__resource__, 'lib').encode("utf-8")).decode("utf-8")
+
+sys.path.append(__resource__)
+sys.path.append(__lib__)
```

Comment:
OK, got it working now - I'd only added init.py to the lib directory, needed it in the resources one as well